### ISSUE TITLE
Bug: nys-fileinput does not have a main focus ring, and doesnt announce file uploads

### ISSUE THREAD

**Issue by esteinborn** [ITS-HCD/nysds#715]
NYS-input should have a focus ring around the main container instead of only the upload files button.

It should also read out that it has had a file uploaded to it as well.

<img width="699" height="362" alt="Image" src="https://github.com/user-attachments/assets/2c35d05e-a83d-46ae-93fb-79dab3b6192e" />

Here is an example from USWDS:

<img width="687" height="490" alt="Image" src="https://github.com/user-attachments/assets/cad2e4bb-4d32-4e8e-b232-e1d8efe85b53" />
